doc.md

```diff
@@ -0,0 +1,110 @@
+# Hackseq 2017
+
+The system allows users to run graphical user interface application (e.g. `igv`), provides 100 GB hard memory and 20 GB soft memory, maps host users to Docker users inside containers and mounts team folders. 
```

Comment:
100 GB hard memory limit and 20 GB soft memory limit

Please describe that there's one user per person and one container per project group.

Reply:
Please start with a few sentences describing…
What is Hackseq?
What is ORCA?
For what purpose is Hackseq using ORCA?

Reply:
The first paragraph is good. For the second, I suggest:

The system provides one container per each project shared by the members of the team, and a user account for each member of the team. Each team shares a project directory, and each user has their own home directory. Graphical user interface applications such as IGV can use X11 through SSH tunnelling.